istioctl/pkg/tag/util.go

```diff
@@ -172,3 +172,48 @@ func DeactivateIstioInjectionWebhook(ctx context.Context, client kubernetes.Inte
 
 	return nil
 }
+
+// DeleteConflictedDefaultTag deletes conflicted tagged MutatingWebhookConfiguration, if it exists
+// and if the default revision MutatingWebhookConfiguration was not deactivated by istioctl.
+func DeleteConflictedDefaultTag(ctx context.Context, client kubernetes.Interface) error {
```

Comment:
I think this can resolve most cases, but it will introduce a new problem: after deleting the installed default webhook, one cannot switch back to the non-revisioned version without re-installing Istio, and after revisioned tag webhook has been removed, there will be no webhook.

Reply:
sounds reasonable, thanks @hanxiaop
I will check your PR today